style.css

```diff
@@ -0,0 +1,154 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
why do you need to put it on everything? it is better to put on specific `div`s when doing so